source/Octopus.Client/Model/OpsProcesses/OpsRunTemplateResource.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace Octopus.Client.Model.OpsProcesses
+{
+    public class OpsRunTemplateResource : DeploymentTemplateResource
```

Comment:
Do these also have promotions? and do we want to use the same property names like `IsDeploymentProcessModified`?

Reply:
Yes for promotions, because tenants are connected to certain environments only (this is slightly different to lifecycles and progression), so this helps us determine what environments you can deploy to during tenanted deployments. And even though we're saying "you can deploy to any environment" in the non-tenanted scenario, I'd like to keep control of that decision server-side via these properties to keep this future proof for this sprint (in case we stumble across a scenario where only certain environments are permitted).

Reply:
And agree, anything "deploymentprocess" related can go in a base class and I'll make separate props for opsSteps.